aggregator/src/aggregator/collection_job_tests.rs

```diff
@@ -417,7 +422,23 @@ async fn collection_job_success_fixed_size() {
         .await;
     assert_eq!(test_conn.status(), Some(Status::BadRequest));
     assert_eq!(
-        take_problem_details(&mut test_conn).await,
+        test_conn
+            .response_headers()
+            .get(KnownHeaderName::ContentType)
+            .unwrap(),
+        "application/problem+json"
+    );
+    let problem_details: serde_json::Value = serde_json::from_slice(
+        &test_conn
+            .take_response_body()
+            .unwrap()
+            .into_bytes()
+            .await
+            .unwrap(),
+    )
+    .unwrap();
+    assert_eq!(
+        problem_details,
```

Comment:
This might be on an outdated branch? This was cleaned up in https://github.com/divviup/janus/commit/a76f06ea539e1ea8b119c8ce54f479e778cd2677 to use utility functions.

Reply:
Yes, this is rebase carnage. I'll clean it up.